db/migrate/20220719131727_create_reports.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReports < ActiveRecord::Migration[6.1]
+  def change
+    create_table :reports do |t|
+      t.string  :title, null: false
+      t.string  :body, null: false
+      t.integer :user_id, null: false
```

Comment:
引数をインデントするなら、第 2 引数以降も揃えないと見づらいので揃えたほうがいいんじゃないかなーと思います。comments についても同様です。

```suggestion
      t.string  :title,   null: false
      t.string  :body,    null: false
      t.integer :user_id, null: false
```

Reply:
修正しました！